packages/backend/src/main/resources/changesets/changeset-1.yml

```diff
@@ -0,0 +1,74 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: kaidobit
+      changes:
+        - createTable:
+            tableName: room
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
We should use modern ids and not this classic auto increment integers.

Like ULID...